### PR TITLE
Feature/upcoming events

### PR DESCRIPTION
**Pull request by SeunghunSunmoonLee**

### CODE REVIEW

File: src/components/UpcomingEvents/index.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  constructor(props) {
+    super(props)
+    this.state = {
+      value: 0,
+    }
+  }
+
+  render() {
+    return (
+      <div id="upcomingEvents" className="upcomingEvents-section">
```
Comment (chutium) [inline src/components/UpcomingEvents/index.js]: where is upcomingEvents-section defined?

**Comment by chutium** [comment]
Reverted token sale box and merged upcoming events.

File: src/components/UpcomingEvents/index.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      </ul>
+    </div>
+  )
+}
+
+export default class UpcomingEvents extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      value: 0,
```
Comment (geastwood) [inline src/components/UpcomingEvents/index.js]: state is not used at all. can be removed, then this component can be refactored to a stateless functional component

File: src/components/UpcomingEvents/index.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+                <FormattedMessage id="upcomingEvents.subtitle" />
+              </p>
+            </Col>
+          </Row>
+        </div>
+        <div className="container">
+          <div className="cards" >
+            {events && events.map(event => {
+              return (
+                <Milestone event={event} />
```
Comment (geastwood) [inline src/components/UpcomingEvents/index.js]: key is missing, will trigger warning in dev mode.

**Pull request merged**